macOS/Configuration/AppStore.xcconfig

```diff
@@ -53,6 +65,12 @@ APP_CONFIGURATION_APP_GROUP[config=Review][sdk=*] = $(APP_CONFIGURATION_BASE_APP
 APP_CONFIGURATION_APP_GROUP[config=Debug][sdk=*] = $(APP_CONFIGURATION_BASE_APP_GROUP).debug
 APP_CONFIGURATION_APP_GROUP[config=Release][sdk=*] = $(APP_CONFIGURATION_BASE_APP_GROUP)
 
+SYSEX_MACH_SERVICE_NAME[sdk=*] = $(NETP_APP_GROUP).ipc
+SYSEX_MACH_SERVICE_NAME[config=CI][sdk=*] = $(NETP_APP_GROUP).ipc
+SYSEX_MACH_SERVICE_NAME[config=Review][sdk=*] = $(NETP_APP_GROUP).ipc
+SYSEX_MACH_SERVICE_NAME[config=Debug][sdk=*] = $(NETP_APP_GROUP).ipc
+SYSEX_MACH_SERVICE_NAME[config=Release][sdk=*] = $(NETP_APP_GROUP).ipc
```

Comment:
This is some basic initial setup.  We'll need more changes to the configuration in order to run the system extension, but I'll take care of those in the next PR where my work will be to let internal users run the VPN as a System Extension.